src/WikiService.Validation/Article/CreateArticleRequestValidator.cs

```diff
@@ -0,0 +1,32 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.WikiService.Data.Interfaces;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests.Article;
+using LT.DigitalOffice.WikiService.Validation.Article.Interfaces;
+
+namespace LT.DigitalOffice.WikiService.Validation.Article
+{
+  public class CreateArticleRequestValidator : AbstractValidator<CreateArticleRequest>, ICreateArticleRequestValidator
+  {
+    public CreateArticleRequestValidator(
+      IArticleRepository articleRepository)
+    {
+
+      CascadeMode = CascadeMode.Stop;
+
+      RuleFor(article => article.RubricId)
+        .NotEmpty().WithMessage("Rubric id must not be empty.")
+        .MustAsync(async (rubricId, _) => await articleRepository.DoesExistAsync(rubricId))
+          .WithMessage("This rubric id does not exist.");
```

Comment:
you check rubric's existence in articles table

Reply:
fixed